less/navigation.less

```diff
@@ -0,0 +1,30 @@
+.header-main {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    padding-top: 1%;
+    padding-bottom: 1%;
+    width: 800px;
+    margin: auto;
+
+    h1 {
+        font-size: 2rem;
+        padding: 1%;
+    }
+}
+
+nav {
+    justify-content: center;
+
+    a {
+        text-decoration: none;
```

Comment:
Great job with the navigation! I would recommend reviewing the design files again to make sure the background color match and to make the navbar items more responsive.